plugins/plugin-compiler.js

```diff
@@ -2,23 +2,32 @@ const path = require('path');
 
 const { createApolloClient, createFile } = require('./util');
 
+const DEFAULT_GRAPHQL_PATH = '/graphql';
+
 class WebpackPlugin {
   constructor(options = {}) {
     this.options = options;
   }
 
   async index(compilation, options) {
-    const { host, plugin } = options;
+    const { url, host, plugin } = options;
+    let endpoint = url;
+
+    if (!endpoint && host) {
+      endpoint = `${host}${DEFAULT_GRAPHQL_PATH}`;
+    }
 
     plugin.outputLocation = path.join(plugin.outputDirectory, plugin.outputName);
 
     console.log(`[${plugin.name}] Compiling file ${plugin.outputLocation}`);
 
-    if (typeof host !== 'string') {
-      throw new Error(`[${plugin.name}] Failed to compile: invalid host type ${typeof host}`);
+    if (typeof url !== 'string' && typeof host !== 'string') {
+      throw new Error(
+        `[${plugin.name}] Failed to compile: invalid url and host type: url ${typeof url}; host ${typeof host}`
+      );
     }
 
-    const apolloClient = createApolloClient(host);
+    const apolloClient = createApolloClient(url);
```

Comment:
The argument shouldn't be `endpoint`?

Reply:
oops good catch. not sure how i didnt get an error with that haha